Test/comp/Extern5Nested.py

```diff
@@ -0,0 +1,13 @@
+"""
+The Python compiler only supports {:extern} code on a module level, so the
+entire module must be supplied.
+"""
+
+import sys, _dafny
+
+assert "Nested.Library" == __name__
+Nested_Library = sys.modules[__name__]
```

Comment:
```suggestion
Library = sys.modules[__name__]
```

Reply:
Why this change, just tidier? I was just trying to be consistent but perhaps we shouldn't have this line at all?